docs/hosting/installation/server-setups/docker-compose.md

```diff
@@ -35,13 +29,12 @@ su - ${USER}
 
 ### 3. Install Docker-Compose
 
-This can vary depending on the Linux distribution used. Before proceeding check the latest version of Docker Compose v1 [on the repository's release page](https://github.com/docker/compose/releases) and replace the `1.29.2` below. Should you wish to use Docker Compose v2 instead you can find detailed instructions [here in the Docker documentation](https://docs.docker.com/compose/cli-command/).
+This can vary depending on the Linux distribution used. You can find detailed instructions [here in the Docker documentation](https://docs.docker.com/compose/).
```

Comment:
```suggestion
This can vary depending on the Linux distribution used. You can find detailed instructions in the [Docker documentation](https://docs.docker.com/compose/){:target=_blank .external-link}.
```